### DISCUSSION TITLE
検索バーのホットキーの提供

### DISCUSSION THREAD

**Discussion by yarakos95** [increments/qiita-discussions#901] [要望・アイデア・不具合]
Google や GitHub 等いくつかのサービスでは、`/` を押下することで検索バーにフォーカスが当たるホットキーが用意されています。

Qiita でも `/` で検索バーにフォーカスが当たるようになれば、サッと Qiita 内の検索が出来るようになって便利です。（割り当てるキーは何でも構いませんが、よく使われている <sup>[要出典]</sup> `/` に合わせてもらえると嬉しいです）

**Comment by tanaka-asami415** [comment]
@Yarakashi-Kikohshi 
ご意見ありがとうございます！
実装されていると便利そうですね。前向きに検討します！
